acceptance/lib/helper.rb

```diff
@@ -193,6 +193,7 @@ def puppet_apply_as_puppet_user
     # installed.
     manifest_path = master.tmpfile("puppetserver_manifest.pp")
     herp_path = master.tmpfile("herp")
+    bin_dir = "/opt/puppetlabs/agent/bin/"
```

Comment:
One final comment...it should be `host['puppetbindir']` rather than `options['puppetbindir']`